.travis.yml

```diff
@@ -5,11 +5,26 @@ language: go
 matrix:
   include:
     - go: 1.4
+      env:
+        - KUBE_TEST_API_VERSIONS=v1
+        - KUBE_TEST_ETCD_PREFIXES=registry
+    - go: 1.3
+      env:
+        - KUBE_TEST_API_VERSIONS=v1
+        - KUBE_TEST_ETCD_PREFIXES=kubernetes.io/registry
 
 install:
+  - env
+  - mkdir $HOME/gopath/src/k8s.io
+  - mv $HOME/gopath/src/github.com/GoogleCloudPlatform/kubernetes/ $HOME/gopath/src/k8s.io
```

Comment:
Given that below we are going to (potentially wrongly) assume that GOPATH only has one entry should we use it here instead of hard coding $HOME/gopath/src ?

mkdir $GOPATH/k8s.io
ln -s -t $GOPATH/k8s.io/ $GOPATH/github.com/GoogleCloudPlatform/kubernetes

And you wouldn't need the "cd"  Then again, I guess you already did this in shippable, so keeping the two close is probably a good thing...